src/mame/snk/hng64_v.cpp

```diff
@@ -833,27 +860,37 @@ uint32_t hng64_state::screen_update_hng64(screen_device &screen, bitmap_rgb32 &b
 			m_videoregs[0x0c],
 			m_videoregs[0x0d]);
 
-	if (0)
-		popmessage("TC: %08x %08x %08x %08x : %08x %08x %08x %08x : %08x %08x %08x %08x : %08x %08x %08x %08x : %08x %08x %08x %08x : %08x %08x %08x %08x",
+	if (1)
+		popmessage("TC: %08x %08x %08x %08x : %08x %08x\n%08x %08x\n%08x %08x\n%08x %08x : %08x %08x %08x %08x : %08x %08x\n%08x %08x : %08x %08x %08x %08x",
```

Comment:
This is going to nuke the popmesage from above – they won’t both be display even if they’re both enabled.

Reply:
yeah these are just for debugging so I can observe the state, I probably should just remember to change them back to 0, they're not enabled for release builds anyway.

Reply:
I suggest using a pattern like https://github.com/mamedev/mame/blob/69380b087a1f44a0795fc20376421b1781bd857b/src/devices/machine/8364_paula.cpp#L225 so that the function body gets compiled even if the `LIVE_TC_VIEW` is zero-ed on top.